src/vm/i386/excepx86.cpp

```diff
@@ -38,6 +38,7 @@
 #include "asmconstants.h"
 #include "virtualcallstub.h"
 
+#ifndef WIN64EXCEPTIONS
```

Comment:
@janvorli I'll move them as you suggested (Actually, it was my first attempt, but I thought that it would make it hard to review).